pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.12"
 datasets = "*"
 pandas = "*"
-litellm = "*"
+litellm = { git = "https://github.com/BerriAI/litellm.git", rev = "58fe6610601297c5a90367fd4583469e2df3fcf9" }
```

Comment:
This was for the tool caching issue, right? I'll remove it when we got the version with it.

Reply:
yep - i'm just waiting for litellm to cut a new release with the fixed cversion